modyn/models/yearbooknet/yearbooknet.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any
+
+import torch
+from torch import nn
+
+
+class YearbookNet:
```

Comment:
(I suppose you will adjust all of these accordingly when introducing the ModynModel with deepcore, right?)

Reply:
Yep, this is the plan